### DISCUSSION TITLE
how to inspect the websocket request?

### DISCUSSION THREAD

**Discussion by GitSleep8** [seleniumbase/SeleniumBase#2433] [General]
Hello Michale, --wire option can inspect the http request, is it possble to inspect the websocket request also?

**Comment by mdmintz** [comment]
selenium-wire is a separate repo: https://github.com/wkeeling/selenium-wire

SeleniumBase has integrations with multiple other projects, such as selenium-wire. For questions specific to those other projects, ask from those GitHub repos.
